### PR TITLE
[ADD] project: project creation and management

### PR DESCRIPTION
**Pull request by toaa-odoo**
Adding a project creation and management page for the project app. Forward to master.

taskid: 3117349

### CODE REVIEW

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hi @jaae-odoo and @toaa-odoo 
Thanks for this, see my comments in the diff

File: content/applications/services/project/project_management.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+.. image:: project_management/project-card.png
+   :align: center
+   :alt: Project card
+
+Add color to project
+~~~~~~~~~~~~~~~~~~~~
+
+A project can be made more recognizable by changing the color of its **card**. To do so, click on
+the :guilabel:`drop-down toggle button (⋮)` and select a color in the bottom-left of the menu.
```
Comment (jaae-odoo) [inline content/applications/services/project/project_management.rst:50]: Add an image

**Comment by jcs-odoo** [comment]
@LoredanaLrpz please have a look at my comments, so that once you approve this PR, you can directly request a review from doc-review.

Cheers :)

File: content/applications/services/project/project_management.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- the :guilabel:`Planned Date` of the project;
+- the total :guilabel:`Allocated Hours` for that project.
+
+.. image:: project_management/project-settings-opened.png
+   :align: center
+   :alt: Project settings
+
+In addition, it is also possible to add a description in the
+:guilabel:`Description` tab, and view related :abbr:`SO (Sales Order)` for that project under the
+:guilabel:`Invoicing` tab. Under the :guilabel:`Settings` tab, an :guilabel:`analytic Account` for
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/services/project/project_management.rst:76]: I found the invoicing tab 
image
but I have no idea why it's there here, why not on other projects... I'd love to discuss this because I'm a bit lost

File: content/applications/services/project/project_management.rst [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+that project can be set, as well as the :guilabel:`Visibility` parameters.
+
+Further options are also available depending on the other **apps** installed on your database.
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:79]: activated and not installed (for Odoo 16.0 and above)

File: content/applications/services/project/project_management.rst (hunk 2 above)
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:76]: I couldn't find analytical account under the settings tab

File: content/applications/services/project/project_management.rst [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+.. note::
+   Users with the **Invoicing** or **Accounting** app installed also have the option to set
+   the project as :guilabel:`Billable`, the **Timesheets** app to log time on tasks within the
+   project, and the **Planning** app to plan resources on tasks within the project.
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:84]: could you please rephrase this? There's too much information, which make the § hard to read

File: content/applications/services/project/project_management.rst [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+Scheduling activities
+=====================
+
+You can schedule **activities** (ex. :guilabel:`Call`, :guilabel:`Meeting`, etc.) per project by
+clicking on the **clock** icon on a project. This opens a list with already scheduled activities,
+and also allows to add **new** activities by clicking :guilabel:`+ Schedule an activity`. In the
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:91]:
```suggestion
clicking on the **clock** icon on a project. Doing so opens a list with already scheduled activities
and allows planning **new** activities by clicking :guilabel:`+ Schedule an activity`. On the pop-up
```

File: content/applications/services/project/project_management.rst [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+pop-up window, select the :guilabel:`Activity Type`, enter a :guilabel:`Summary` for that activity,
+a :guilabel:`Due Date`, and :guilabel:`Assigned to` employee. According to the :guilabel:`Activity
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:93]: Can you rephrase this part:  and :guilabel:`Assigned to` employee

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
Hello @jaae-odoo , I feel like it's hard to talk about projects' settings since they change drastically depending on the apps activated, so I'd love to talk about your pr IRL tomorrow if you have time.

Also, this isn't used and should be deleted: content/applications/services/project/project_management/project-activities.png

File: content/applications/services/project/project_management.rst [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+:guilabel:`Projects`.
+
+.. seealso::
+   `Odoo Tutorials: Customize your project
+   <https://www.odoo.com/slides/slide/customize-your-project-1662?fullscreen=1>`_
+
+Settings
+--------
+
+To access the settings of a project, click on the :guilabel:`drop-down toggle button (⋮)` of the
```
[the remark this answers is not in the document]
Comment (jaae-odoo) [reply content/applications/services/project/project_management.rst:56]: Remove the line altogether

File: content/applications/services/project/project_management.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (jaae-odoo) [reply content/applications/services/project/project_management.rst:76]: say that there has to be a sales order linked to the project

Comment (jaae-odoo) [reply content/applications/services/project/project_management.rst:76]: download accounting app and activate analytics in the configurations

Comment (jaae-odoo) [reply content/applications/services/project/project_management.rst:76]: or link to existing doc in accounting

File: content/applications/services/project/project_management.rst (hunk 6 above)
Comment (jaae-odoo) [reply content/applications/services/project/project_management.rst:93]: and assign the project to an employee

File: content/applications/services/project/project_management.rst [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+   `Odoo Tutorials: Customize your project
+   <https://www.odoo.com/slides/slide/customize-your-project-1662?fullscreen=1>`_
+
+Depending on the apps you have activated, other options are available:
+
+- **Sales orders**: you can view related :abbr:`SO (Sales Order)` for that project under the
+  :guilabel:`Invoicing` tab by activating the **Invoicing** app;
+
+- **Billable**: you can set the project as :guilabel:`Billable` by activating the **Invoicing** or
+  **Accounting** apps;
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:69]: I think it's important to mention the sections under which these settings can be found, and also what they do; maybe like:
- Section
  - :guilabel:`Setting` (the written name): small explanation
  
or something like that: 

- You can enable the :guilabel:`Billable` setting to invoice customers with time spent and materials used under the :guilabel:`Sales & Invoicing` section by activating the Invoicing or Accounting apps;

also, you don't mention other settings, so why some and not others?
image

File: content/applications/services/project/project_management.rst [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+- **Log time**: you can log time on tasks by activating the **Timesheet** app;
+
+- **Plan resources**: you can plan resources on tasks by activating the **Planning** app;
+
+- **Analytic account**: you can set an :guilabel:`Analytic Account` for that project if you have
+  the **Accounting** app activated as well as the *Analytic Accounting* option in the
+  configurations.
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:77]: weird to talk about it in the end when it's the first setting you see in the settings tab,
also, you don't talk about "visibility", but I think you do not need to activate any app to do it (same for milestones etc.)
So I suggest dividing this in two: the settings available in the settings tab independently of the apps activated, and the settings that depend on the apps activated

File: content/applications/services/project/project_management.rst [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+.. seealso::
+   `Analytic account use cases
+   <https://www.odoo.com/documentation/16.0/applications/finance/accounting/others/analytic/usage.html>`_
+
+Scheduling activities
+=====================
+
+You can schedule **activities** (ex. :guilabel:`Call`, :guilabel:`Meeting`, etc.) per project by
+clicking on the **clock** icon on a project. doing so opens a list with already scheduled activities
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:87]: project. Doing

File: content/applications/services/project/project_management.rst [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+
+- **Reporting**: analyze your project's progress and profitability through graphics and statistics;
+
+- **Color**: make a line of color appear on the left side of the card so that your project is more
+  recognizable;
+
+- **Settings**: you can change the following:
+
+  - the :guilabel:`Name` of the project;
+  - the :guilabel:`Name of the tasks` found under that project.
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:44]: that project;

File: content/applications/services/project/project_management.rst [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+.. image:: project_management/project-settings.png
+   :align: center
+   :alt: Project card
+
+This enables a new menu divided into four parts:
+
+- **View**: see an overview of your project's components, such as its tasks, milestones and project
+  updates. Depending on which apps you have activated, more options may be available, such as
+  **Documents**: all uploaded files can be found under this menu, as well as in the **Documents**
```
Comment (LoredanaLrpz) [inline content/applications/services/project/project_management.rst:33]: I'm not loving this : inside a : 
image
but that's nitpicking

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
Hello @jaae-odoo , here's the review :)

**Review by LoredanaLrpz — APPROVED** [review]
@jaae-odoo hey, I force-pushed because there was only two indentation issues, so it's approved

File: content/applications/services/project/project_management.rst [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+.. image:: project_management/project-settings.png
+   :align: center
+   :alt: Project card
+
+This enables a new menu divided into four parts:
+
+- :guilabel:`View`: see an overview of your project's components, such as its :guilabel:`tasks`,
+  :guilabel:`milestones` and :guilabel:`project updates`. Depending on which apps you have
+  activated, more options may be available, such as :guilabel:`Documents`` All uploaded files can be
```
Comment (AntoineVDV) [inline content/applications/services/project/project_management.rst:33]:
```suggestion
  activated, more options may be available, such as :guilabel:`Documents` All uploaded files can be
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+ delegate=LoredanaLrpz

**Comment by LoredanaLrpz** [comment]
@robodoo r+
